publish-service/src/test/java/com/ericsson/eiffel/remrem/publish/service/EventTemplateHandlerTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+    Copyright 2018 Ericsson AB.
+    For a full list of individual contributors, please see the commit history.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+package com.ericsson.eiffel.remrem.publish.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.io.FileUtils;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.io.File;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+public class EventTemplateHandlerTest {
+
+    private static Logger log = LoggerFactory.getLogger(EventTemplateHandlerTest.class);
+    static private final String inputFilePathData = "src/test/resources/testDataForParsingEvents/";
+    static private final String inputFilePathExpectedData = "src/test/resources/expectedParsedEvents/";
```

Comment:
both these variables rename with uppercases (java convention) and in order -> private static final